paws_collector.js

```diff
@@ -106,6 +111,7 @@ class PawsCollector extends AlAwsCollector {
         this._pawsEndpoint = process.env.paws_endpoint
         this._pawsDomainEndpoint = endpointDomain;
         this._pawsHttpsEndpoint = 'https://' + endpointDomain;
+        this.ddbTableName = process.env.pawsDDBTableName;
```

Comment:
lets' use `_pawsDdbTableName`, to keep it consistent with the rest of properties